tests/fixtures/homematicip_cloud.json

```diff
@@ -14,6 +14,345 @@
         }
     },
     "devices": {
+        "3014F7110000000000ABCD50": {
```

Comment:
Is this info really used by home assistant directly? Couldn't we replace this by patching the library interfaces?

Reply:
Yes it is used by HA. These identifiers are already anonymized.
Why do you want to patch here?

Reply:
It's part of the unique_id.

Reply:
I mean that we can replace all of this fixture info by patching the library interfaces to a greater extent. I assume this fixture info is a representation of a response by a the cloud api and stored in the library client. I assume we aren't using the info directly in home assistant but the client is using it to create objects that represent the info. Is that correct?

Reply:
Yes, your assumtion is correct. 
But this is exactly the same like they do with hue and deconz. They create a json response that normally returned by the bridge, so why should i change here? It's quiet simple to create these static test data, that will be manipulated during the test.

Reply:
We could avoid keeping this file updated by patching the library client methods that we use in home assistant to return the correct objects.

This fixture contains info that home assistant isn't directly interested in so we try to avoid taking responsibility for that info. It's the same reasoning as we want the device interface code to be contained within a 3rd party library.

Reply:
Anyway, this is more an observation and not a change request.